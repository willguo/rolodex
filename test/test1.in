Booker T., Washington, 87360, 373 781 7380, yellow
Chandler, Kerri, (623)-668-9293, pink, 123123121
James Murphy, yellow, 83880, 018 154 6474
asdfawefawea