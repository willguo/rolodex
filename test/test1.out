{
  "entries": [
    {
      "color": "yellow", 
      "firstname": "James", 
      "lastname": "Murphy", 
      "phonenumber": "018-154-6474", 
      "zipcode": "83880"
    }, 
    {
      "color": "yellow", 
      "firstname": "Booker T.", 
      "lastname": "Washington", 
      "phonenumber": "373-781-7380", 
      "zipcode": "87360"
    }
  ], 
  "errors": [
    1, 
    3
  ]
}